### ISSUE TITLE
Power isolation needed when wireless

### ISSUE THREAD

**Issue by ajfisher** [nodebotsau/simplebot#3]
At the moment the assumption is that the servos run off 4xAA batteries (ish 6v) and that any arduino is tethered and as such it, plus any sensors run off 5V usb.

I noticed at CampJS that when running an arduino + motors + bluetooth all off the battery pack that current draw on the servos would occasionally reset the arduino. This obviously got worse as the battery discharged. 

Suggestion then on having 2 supplies when in "wireless" mode to supply the MCU + sensors and then the main pack for the servos / motors. Other than that, doing some kind of supply filter stuff to smooth out any drops.

Would be good to get opinions on this; ping: @nog3 @geekscape

**Comment by devgeeks** [comment]
:+1: 

I think this is why, once I connected the wireless to my simplebot, it would only respond for a very short time before (I assume) resetting.

**Comment by ajfisher** [comment]
Yep I think for next time we'll have a look at some other battery
options... We've always shied away from lipos because peeps don't generally
have balancing chargers but we'll see what paths we can go down.
